docs/UserGuide.md

```diff
@@ -84,21 +84,58 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Creating tags: `create`
+
+Creates a tag and categorises it to the specified category.
+You can customize these categories as per your needs and tags will be 
+color coded by the different categories. 
+
+The only restriction is you can only define up to 5 different categories.
+
+Format: `create t/CATEGORY NAME…​`
```

Comment:
I think doing `create t/CATEGORY TAG` or `create t/CATEGORY TAGNAME` would be clearer